includes/OpenAI/OpenAI_AI_Service.php

```diff
@@ -137,8 +137,13 @@ static function ( array $model ) {
 		return array_reduce(
 			$model_slugs,
 			static function ( array $model_caps, string $model_slug ) use ( $gpt_capabilities, $gpt_multimodal_capabilities ) {
-				if ( str_starts_with( $model_slug, 'gpt-' ) && ! str_contains( $model_slug, '-instruct' ) ) {
-					if ( str_starts_with( $model_slug, 'gpt-4' ) ) {
+				if (
+					str_starts_with( $model_slug, 'gpt-' )
+					&& ! str_contains( $model_slug, '-instruct' )
+					&& ! str_contains( $model_slug, '-realtime' )
+					&& ! str_contains( $model_slug, '-audio' )
+                ) {
```

Comment:
WPCS requires tabs for indentation:
```suggestion
					&& ! str_contains( $model_slug, '-audio' )
				) {
```